baseband/vlbi_base/base.py

```diff
@@ -188,19 +203,19 @@ def tell(self, unit=None):
         if unit == 'time':
             return self.start_time + self.tell(unit=u.s)
 
-        return (self.offset * u_sample).to(unit, equivalencies=[(u.s, u.Unit(
-            self.samples_per_frame * self.frames_per_second * u_sample))])
+        return (self.offset / self.sample_rate).to(unit)
 
     def _frame_info(self):
         offset = (self.offset +
                   self.header0['frame_nr'] * self.samples_per_frame)
+        framerate = self.sample_rate.to_value(u.Hz) / self.samples_per_frame
```

Comment:
This is really VLBI only so we can assume the framerate itself is integer, i.e.,
```
frames_per_second = int(round((self.sample_rate / self.samples_per_frame).to(u.Hz)))
```
and then we no longer have to cast anything else to `int` below.

Reply:
OK as per the comment above, but for each maybe just `to_value(u.Hz)` to avoid the issue with `round`